utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/json/swagger/extractor/JsonTypedMap.scala

```diff
@@ -0,0 +1,21 @@
+package pl.touk.nussknacker.engine.json.swagger.extractor
+
+import io.circe.syntax.EncoderOps
+import io.circe.{Json, JsonObject}
+import pl.touk.nussknacker.engine.json.swagger._
+import pl.touk.nussknacker.engine.json.swagger.extractor.JsonToNuStruct.JsonToObjectError
+
+import pl.touk.nussknacker.engine.util.cache.LazyMap
+
+import scala.jdk.CollectionConverters._
+
+final case class JsonTypedMap(jsonObject: JsonObject, definition: SwaggerObject, path: String = "")
```

Comment:
Let's add @TypeInfo to accelerate Kryo serialization ;)

Reply:
Probably we can do the same with SwaggerObject and other types, WDYT @piotrp ?

Reply:
We need a test that puts this through Flink's serialization+deserialization, to verify that it doesn't break data when actually used.

Also, can you create a benchmark that compares performance of serialization+deserialization round-trip with TypedMap (or just HashMap)?

Reply:
To verify this, test should use parallelism > 1?

Reply:
That won't do anything if you won't force Flink to generate two tasks - this can be done with `.shuffle`, `keyBy` etc. But a simpler solution would be to just grab `Serializer` from generated `TypeInformation`.